packages/react/src/primitives/types/textArea.ts

```diff
@@ -2,9 +2,10 @@ import { Property } from 'csstype';
 
 import { Sizes } from './base';
 import { FieldVariations } from './field';
+import { BaseStyleProps } from './style';
 import { ViewProps } from './view';
 
-export interface TextAreaStyleProps {
+export interface TextAreaStyleProps extends BaseStyleProps {
```

Comment:
I don't think this is necssary, because ViewProps already [extends BaseStyleProps](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/src/primitives/types/view.ts#L74).